core/src/main/scala/com/broadinstitute/dsp/models.scala

```diff
@@ -30,3 +31,9 @@ final case class InitBucketToRemove(googleProject: GoogleProject, bucket: Option
 
 final case class K8sClusterToScan(id: Long, kubernetesClusterId: KubernetesClusterId)
 final case class NodepoolToScan(id: Long, nodepoolId: NodepoolId)
+
+final case class KubernetesClusterToRemove(id: Long, googleProject: GoogleProject)
+// TODO: 'project' below is unnecessary but removing it requires an accompanying change in back Leo so leaving for now
+final case class DeleteKubernetesClusterMessage(clusterId: Long, project: GoogleProject, traceId: Option[TraceId]) {
```

Comment:
minor, can you make this `TraceId` non optional. also I would probably move this to `KubernetesClusterRemover` file instead of puttng it in shared model file